### DISCUSSION TITLE
installed but doesnt do anything

### DISCUSSION THREAD

**Discussion by ljmerza** [yoavbls/pretty-ts-errors#56] [Q&A] [answered]
hello, i installed this extension in my vscode but when pulling up ts or tsx files with ts errors, i still get the "unpretty" format for the ts errors. I dont see any config so not sure where to go from here

**Comment by yoavbls** [comment]
Hey @ljmerza, can you write what is the extension version and the version of your VSCode, please? A Screenshot also be helpful 🙏🏼

**Comment by ljmerza** [comment, marked as answer]
hey it started working and i havent had any problems. great extension thanks!
